hedera-node/src/main/java/com/hedera/services/store/TypedTokenStore.java

```diff
@@ -271,24 +245,30 @@ public void persistToken(Token token) {
 		mapModelChangesToMutable(token, mutableToken);
 
 		if (token.hasMintedUniqueTokens()) {
-			final var currentNfts = nfts.get();
+			final var currentNfts = uniqueTokens.get();
+			final var currentUniqueTokenAssociations = uniqueTokenAssociations.get();
+			final var currentUniqueTokenAccountOwnerships = uniqueOwnership.get();
```

Comment:
I think that we can skip the variable declaration and use directly the `uniqueTokens.get();` method as-well as `uniqueTokenAssociations.get();` and `uniqueOwnership.get();`

Same thing for `burn`